src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -43,31 +43,19 @@ object DUOS {
     def apply(jsValues: Map[String, JsValue]): DuosDataUse = {
       def getBoolean(f: String): Option[Boolean] = {
         jsValues.get(f) match {
-          case x if x.isDefined =>
-            Try(jsValues(f).convertTo[Boolean]) match {
-              case Success(booleanVal) => Some(booleanVal)
-              case _ => None
-            }
+          case Some(b: JsBoolean) => Try(b.convertTo[Boolean]).toOption
           case _ => None
         }
       }
       def getSeqString(f: String): Option[Seq[String]] = {
         jsValues.get(f) match {
-          case x if x.isDefined =>
-            Try(jsValues(f).convertTo[JsArray].elements.map(_.convertTo[String])) match {
-              case Success(listVal) => Some(listVal)
-              case _ => None
-            }
+          case Some(l: JsArray) => Try(l.elements.map(_.convertTo[String])).toOption
```

Comment:
I still like using match/case/collect here to look for Strings and skip other types, instead of Try-ing to convert to a string and using exceptions to handle non-strings. But I don't feel too strongly about this and won't hold the PR for that.

Reply:
Ah - we are doing that in the serialization, I didn't port that over to the deserialization.